examples/column/fct_advection.jl

```diff
@@ -76,50 +78,58 @@ domain = Domains.IntervalDomain(
-zc = Fields.coordinate_field(cs)
-
-# Unitary, constant advective velocity
-w = Geometry.WVector.(speed .* ones(FT, fs))
-
-# Define a pulse wave or square wave
-pulse(z, t, z₀, zₕ, z₁) = abs(z.z - speed * t) ≤ zₕ ? z₁ : z₀
-
-# Initial condition
-y0 = pulse.(zc, 0.0, z₀, zₕ, z₁)
-
-# Set up parameters needed for time-stepping
-Δt = 0.0001
-dydt = copy(y0)
-corrected_antidiff_flux = similar(y0)
-parameters = (; w, C, corrected_antidiff_flux)
-# Call the RHS function
-f!(dydt, y0, parameters, t₀)
-t₁ = 100Δt
-prob = ODEProblem(f!, y0, (t₀, t₁), parameters)
-sol = solve(
-    prob,
-    SSPRK33(),
-    dt = Δt,
-    saveat = Δt,
-    progress = true,
-    adaptive = false,
-    progress_message = (dt, u, p, t) -> t,
-)
-computed_result = sol.u[end]
-analytical_result = pulse.(zc, t₁, z₀, zₕ, z₁)
-err = norm(computed_result .- analytical_result)
-initial_mass = sum(sol.u[1])
-mass = sum(sol.u[end])
-rel_mass_err = norm((mass - initial_mass) / initial_mass)
-
-@test err ≤ 0.018
-@test rel_mass_err ≤ eps()
-
-plot(sol.u[end])
-Plots.png(
-    Plots.plot!(analytical_result, title = "FCT (C=0.5)"),
-    joinpath(path, "exact_and_computed_advected_square_wave_C_05.png"),
-)
+stretch_fns = (Meshes.Uniform(), Meshes.ExponentialStretching(FT(7.0)))
+err = zeros(FT, length(stretch_fns))
+initial_mass = zeros(FT, length(stretch_fns))
+mass = zeros(FT, length(stretch_fns))
+rel_mass_err = zeros(FT, length(stretch_fns))
+
+for (i, stretch_fn) in enumerate(stretch_fns)
+
+    mesh = Meshes.IntervalMesh(domain, stretch_fn; nelems = n)
+    cs = Spaces.CenterFiniteDifferenceSpace(mesh)
+    fs = Spaces.FaceFiniteDifferenceSpace(cs)
+    zc = Fields.coordinate_field(cs)
+
+    # Unitary, constant advective velocity
+    w = Geometry.WVector.(speed .* ones(FT, fs))
+
+    # Define a pulse wave or square wave
+    pulse(z, t, z₀, zₕ, z₁) = abs(z.z - speed * t) ≤ zₕ ? z₁ : z₀
+
+    # Initial condition
+    y0 = pulse.(zc, 0.0, z₀, zₕ, z₁)
+
+    # Set up fields needed for time-stepping
+    dydt = copy(y0)
+    corrected_antidiff_flux = similar(y0)
+    parameters = (; w, C, corrected_antidiff_flux)
+    # Call the RHS function
+    f!(dydt, y0, parameters, t₀)
+
+    prob = ODEProblem(f!, y0, (t₀, t₁), parameters)
+    sol = solve(
+        prob,
+        SSPRK33(),
+        dt = Δt,
+        saveat = Δt,
+        progress = true,
+        adaptive = false,
+        progress_message = (dt, u, p, t) -> t,
+    )
+    computed_result = sol.u[end]
+    analytical_result = pulse.(zc, t₁, z₀, zₕ, z₁)
+    err[i] = norm(computed_result .- analytical_result)
+    initial_mass[i] = sum(sol.u[1])
+    mass[i] = sum(sol.u[end])
+    rel_mass_err[i] = norm((mass[i] - initial_mass[i]) / initial_mass[i])
+
+    @test err[i] ≤ 0.11
+    @test rel_mass_err[i] ≤ 3eps()
+
+    plot(sol.u[end])
+    Plots.png(
+        Plots.plot!(analytical_result, title = "FCT (C=0.5)"),
+        joinpath(path, "exact_and_computed_advected_square_wave_C_05.png"),
+    )
```

Comment:
Just a heads up, this looks like it will overwrite `exact_and_computed_advected_square_wave_C_05.png` for all `for (i, stretch_fn) in enumerate(stretch_fns)`

Reply:
Yes, I thought about it and I wasn't sure if we wanted to preserve both to be honest

Reply:
I just pushed a change to preserve both artifacts